es6-classes/example-class.js

```diff
@@ -0,0 +1,22 @@
+class ExampleClass {
+  constructor(example) {
+    this.example = example;
+  }
+
+  getExample() {
+    return this.example;
+  }
+}
```

Comment:
Please make sure you don't add any new functionality to the `class` that wasn't in the original `function` constructor.